docs/context/ref/Action.md

```diff
@@ -6,12 +6,11 @@ hide_title: true
 ---
 # `Action`
```

Comment:
this seems to be missing front matter to allow it to appear in the docusaurus nav.

Reply:
schema linking otherwise looks good though.

Reply:
I just looked at the preview

Reply:
That'll be a missing sidebar entry then, will add to PR - thanks for flagging it

Reply:
Should be fixed - spotted a couple of other minor errors in the sidebar and fixed those too.